### DISCUSSION TITLE
Correct syntax for alt text?

### DISCUSSION THREAD

**Discussion by imbwaldo** [spatie/statamic-responsive-images#139] [Q&A]
@riasvdv, what is the correct HTML syntax for pulling alt text from the Statamic Asset Manager?

**Comment by ncla** [comment]
Hey,

Since v2.12.4 you should not need to do anything special to pull alt text from an asset. Just do `{{ responsive:asset }}` and you should be good to go.

**Reply by imbwaldo** [reply]
Thanks, @ncla. And thanks again for getting this fixed.
